cmd/ctr-remote/sampler/sampler.go

```diff
@@ -29,8 +30,14 @@ import (
 	"time"
 
 	"github.com/containerd/containerd/log"
+	"github.com/containerd/containerd/pkg/netns"
+	gocni "github.com/containerd/go-cni"
 	runc "github.com/containerd/go-runc"
 	"github.com/docker/docker/oci"
+	"github.com/docker/libnetwork/etchosts"
+	"github.com/docker/libnetwork/resolvconf"
```

Comment:
Do you really need libnetwork dependencies?

Reply:
Thanks for the comment. Updated not to use libnetwork.